android/automation.md

```diff
@@ -0,0 +1,175 @@
+# Automation and continuous integration systems
+
+Our Android projects share a similar automation model:
+- Taskcluster is our primary CI: we use it for as much automation as
+possible, including our core builds and releases, because it's a
+flexible first party service we can rely on.
+- Travis is secondary: we use it to get automation working quickly or get
+support for tools that won't work on Taskcluster.
```

Comment:
All tools should work on taskcluster too. In fact I would like to get rid of travis in a-c.

Reply:
Changed to:
> - Travis is secondary: we use it to get some automation tools working
quickly but all tools should eventually migrate to Taskcluster.

If you want, I can remove the whole thing so no one is encouraged to use it further!